tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -149,6 +149,12 @@ bgp/test_bgp_bbr.py:
       - "platform in ['x86_64-8111_32eh_o-r0', 'x86_64-8122_64eh_o-r0', 'x86_64-8122_64ehf_o-r0']"
       - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/17598"
 
+bgp/test_bgp_command.py:
```

Comment:
we have fixed many service port related topo fixes. this issue should not appear anymore (not seeing any hits from both side data)

Reply:
just found that issue is also closed.